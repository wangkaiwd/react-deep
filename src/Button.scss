.button {
  font-size: 40px;
  color: red;
}
