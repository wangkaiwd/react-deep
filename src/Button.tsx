import React from 'react';
// import '@/Button.scss';
const Button: React.FC = () => {
  return (
    <div className="button">
      I am a button
    </div>
  );
};

export default Button;
