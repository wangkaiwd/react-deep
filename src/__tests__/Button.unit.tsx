import renderer from 'react-test-renderer';
import Button from '@/Button';
import * as React from 'react';

describe('Button', () => {
  it('是个div', () => {
    const json = renderer.create(<Button/>).toJSON();
    expect(json).toMatchSnapshot();
  });
});
