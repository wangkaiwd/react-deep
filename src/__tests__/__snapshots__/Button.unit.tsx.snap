// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button 是个div 1`] = `
<div
  className="button"
>
  I am a button
</div>
`;
